README.md

```diff
@@ -4,39 +4,16 @@ Running [`scripts/start`](scripts/start) will start an Elasticsearch instance,
 loading data from
 [data/input/gp-data-merged.json](data/input/gp-data-merged.json) into an index
 named `profiles` [localhost:9200/profiles](http://localhost:9200/profiles).
+When running the container in this way, the port on which ES is exposed can be
+overridden by setting the environment variable `ES_PORT` (e.g. `ES_PORT=9201 ./scripts/start`).
 
-Sample query to return the top 3 of a query by GP name:
-```
-curl -s -XPOST "http://localhost:9200/profiles/_search?pretty" -d '
-{
-  "size": 3,
-  "explain": false,
-  "_source": [ "name", "address", "doctors" ],
-  "query" : { 
-    "multi_match": { 
-      "query": "Doctor O'Connor",
-      "fields": [ "name^2", "address.addressLines", "doctors" ],
-      "operator": "or"
-    }
-  },
-  "highlight" : {
-    "fields" : {
-      "name" : {},
-      "doctors" : {},
-      "address.addressLines" : {}
-    }
-  }
-}'
-```
+A basic set of tests can be run from [`scripts/test`](scripts/test)
 
-When running the container in this way, the port on which ES is exposed can be
-overridden by setting an environment variable - `ES_PORT` - equal to the port
-wishing to be used e.g. `ES_PORT=9201 ./scripts/start`.
+There are a number of sample queries in [sample-queries](./sample-queries) which can be used to query the data. (e.g  `curl -s -XPOST "http://localhost:9200/profiles/_search?pretty" -d @sample_queries/query-nested-doctor.json | jq -C '.' | less -R`)
```

Comment:
I'm a little unclear as to the point of the sample queries as they aren't all doing what `gp-finder` is doing. Can a `README` be added to the `sample-queries` directory explaining what each one is there for  and perhaps when each query might be useful to use?